File: src/Button/MyFrame.java
    package Button;

    import javax.swing.*;
    import java.awt.*;
    import java.awt.event.ActionEvent;
    import java.awt.event.ActionListener;

    public class MyFrame  extends JFrame implements ActionListener {
        JButton button;
        JLabel label;
        MyFrame() {
            ImageIcon icon = new ImageIcon("src/Button/icon.png");
            ImageIcon icon2 = new ImageIcon("src/Button/Logo.png");
            label = new JLabel();
            label.setIcon(icon2);
            label.setBounds(150, 300, 150, 150);
            label.setVisible(false);

            button = new JButton();
            button.setBounds(100, 100, 250, 100);
            button.setText("Aku Adalah tombol");
            button.setFocusable(false);
            button.setIcon(icon);
            button.setHorizontalTextPosition(JButton.CENTER);
            button.setVerticalTextPosition(JButton.BOTTOM);
            button.setFont(new Font("Comic Sans", Font.PLAIN, 14));
            button.setIconTextGap(-200);
            button.setForeground(Color.cyan);
            button.setBorder(BorderFactory.createEtchedBorder());
    //        button.setEnabled(false);
            button.addActionListener(this);
            button.addActionListener(e -> {
                System.out.println("poo");
            });
            this.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
            this.setLayout(null);
            this.setSize(500, 500);
            this.setVisible(true);
            this.add(button);
            this.add(label);
        }

        @Override
        public void actionPerformed(ActionEvent e) {
            if(e.getSource() == button) {
                label.setVisible(true);
            }
        }
    }

File: src/CheckBox/MyFrame.kt
package CheckBox

import java.awt.FlowLayout
import java.awt.Font
import java.awt.event.ActionEvent
import java.awt.event.ActionListener
import javax.swing.*

class MyFrame : JFrame(), ActionListener {
     val button = JButton()
    val checkBox = JCheckBox()
    var xIcon = ImageIcon()
    var check = ImageIcon()
    init {
       defaultCloseOperation = EXIT_ON_CLOSE
        layout = FlowLayout()

        xIcon = ImageIcon("src/CheckBox/x.png")
        check = ImageIcon("src/CheckBox/v.png")


        button.setText("Submit")
        button.addActionListener(this)
        add(button)

        checkBox.setText("I'm not a robot")
        checkBox.setFocusable(false)
        checkBox.font = Font("Menlo", Font.PLAIN, 20)
        checkBox.setIcon(xIcon)
        checkBox.setSelectedIcon(check)
        add(checkBox)



        pack()
        isVisible = true
    }

    override fun actionPerformed(e: ActionEvent?) {
        if(e?.source == button)
         println(checkBox.isSelected)



    }
}